src/applications/public-outreach-materials/sass/public-outreach-materials.scss

```diff
@@ -0,0 +1,111 @@
+$formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
+
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-nav-sidebar";
+
+.outreach-asset-form-filters {
+  padding: 15px;
+  background: $color-gray-lightest;
+  max-width: 95%;
+  margin-bottom: 25px;
+  @media (max-width: $medium-screen) {
+    .vads-l-col--6,
+    .vads-l-col--5,
+    .vads-l-col--4 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  .vads-l-grid-container {
+    padding-left: 0;
+    padding-right: 0;
+  }
+  label {
+    margin-top: 0;
+  }
+  select {
+    max-width: 100%;
+  }
+}
+
+.asset-component-library {
+  @media (max-width: $medium-screen) {
+    .vads-l-col--6 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  .asset-card {
+    min-height: 450px;
+    float: left;
+    display: none;
+    &.show-type {
```

Comment:
I think if you're able to use just the `hide-` variant of these classes like I suggested above, these styles would get simpler and you could just have:

```css
.hide-topic, .pager-hide, .hide-type {
  display: none !important;
}
```

Reply:
I don't think this will work - the above styles take into account combined filtering. If there was only one filter, then this approach would be enough, but with two, I need to nest classes, @jbalboni